src/GNNGraphs/transform.jl

```diff
@@ -694,6 +694,25 @@ function rand_edge_split(g::GNNGraph, frac; bidirected = is_bidirected(g))
     return g1, g2
 end
 
+function add_RandomWalkPE!(g::GNNGraph, walk_length::Int, symbol::Symbol=:p)
+    matrix=zeros(walk_length,g.num_nodes)
+    adj = adjacency_matrix(g)
```

Comment:
```suggestion
    adj = adjacency_matrix(g, Float32; dir=:out)
```
`dir=:out` is the default (meaning A[i,j] represents the edge from i to j) but better be explicit here for clarity